pkgs/development/python-modules/debugpy/default.nix

```
@@ -66,7 +66,7 @@ buildPythonPackage rec {
       "x86_64-darwin"  = "-std=c++11 -lc -D_REENTRANT -dynamiclib -o attach_x86_64.dylib";
       "i686-darwin"    = "-std=c++11 -lc -D_REENTRANT -dynamiclib -o attach_x86.dylib";
       "aarch64-darwin" = "-std=c++11 -lc -D_REENTRANT -dynamiclib -o attach_arm64.dylib";
-    }.${stdenv.hostPlatform.system} or (throw "Unsupported system: ${stdenv.hostPlatform.system}")}
+    }.${stdenv.hostPlatform.system} or (throw "Unsupported system: ${stdenv.hostPlatform.system} for package ${pname}")}
```

An "attribute missing" error should be fine if someone sets `NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1`.
```suggestion
    }
```

---

python's disable in python-packages is rather dirty and if we remove the or here we error with a syntax error when the platform is not supported.

---

It not about `disabled`, this is about `meta.platforms`.